vcell-client/src/main/java/cbit/vcell/client/TranslationLogger.java

```diff
@@ -93,10 +94,10 @@ public void sendAllMessages() {
 
 	@Override
 	public void sendMessage(Priority p, ErrorType et, String message)
-			throws Exception {
+			throws VCLoggerException {
 
 		if  (p == null ||et == null ) {
-			throw new IllegalArgumentException("Invalid params for sending translation message.");
+			throw new VCLoggerException(new IllegalArgumentException("Invalid params for sending translation message."));

```

Comment:
Comment:  you don't have to wrap subclasses of RuntimeException in VCLoggerException if it doesn't add clarity of intent or control flow.  Recall that RuntimeExceptions never have to be declared.  Your call ...